ProcessMaker/Repositories/CaseUtils.php

```diff
@@ -31,72 +31,105 @@ public static function getCaseNumberByKeywords(int $caseNumber)
     /**
      * Store processes.
      *
-     * @param ExecutionInstanceInterface $instance
      * @param Collection $processes
+     * @param array|null $processData
+     *     An optional array of additional process data. Each element in the array should have the following structure:
+     *     [
+     *         'id' => int,    // The unique identifier of the process
+     *         'name' => string  // The name of the process
+     *     ]
+     * @param string|null $processName
      * @return Collection
      */
-    public static function storeProcesses(ExecutionInstanceInterface $instance, Collection $processes)
+    public static function storeProcesses(Collection $processes, ?array $processData = []): Collection
```

Comment:
maybe we can add unit test for these methods

Reply:
there is a ticket for it https://processmaker.atlassian.net/browse/FOUR-19514